packages/orchestration/src/examples/stakeAtom.contract.js

```diff
@@ -47,17 +47,17 @@ export const start = async (zcf, privateArgs, baggage) => {
     zcf,
   );
 
-  async function createAccount() {
-    const account = await E(orchestration).createAccount(
+  async function makeAccount() {
+    const account = await E(orchestration).makeAccount(
       hostConnectionId,
       controllerConnectionId,
     );
-    const accountAddress = await E(account).getAccountAddress();
-    trace('account address', accountAddress);
+    const address = await E(account).getAddress();
```

Comment:
btw feel free to name the variable `accountAddress` if that seems more informative

Reply:
👍 going to keep it as `address` for consistency. I might lobby to change `getRemoteAddress()` to `getRemoteChannelAddress()` (and `getLocalAddress()` -> `getLocalChannelAddress()`) so no one gets confused by these.

Reply:
On second thought, since I am going to change things to use `ChainAddress`, i will call it `chainAddress`. Was awkward typing `this.state.address.address` to form a message.